src/Umbraco.Web.BackOffice/Extensions/UmbracoApplicationBuilder.BackOffice.cs

```diff
@@ -30,6 +31,7 @@ public static IUmbracoMiddlewareBuilder WithBackOffice(this IUmbracoMiddlewareBu
                 a => a.UseMiddleware<KeepAliveMiddleware>());
 
             builder.AppBuilder.UseMiddleware<BackOfficeExternalLoginProviderErrorMiddleware>();
+            builder.AppBuilder.UseMiddleware<BasicAuthAuthenticationMiddleware>();
```

Comment:
Should this be in `WithBackOffice`?  Really it's running to protected public access to the front-end websites, so does it make more sense in `WithWebsite()`?

Reply:
make sense.. but requires we move the IBackOfficeSignInManager to the common project

Reply:
That's seems OK to me if it's just the interface.  Raises another thought though in that if there's a long-term aim to be able to run Umbraco just with `WithWebsite()` - for front-end only installations - you might also want to change this:

```
    IBackOfficeSignInManager backOfficeSignInManager =
        context.RequestServices.GetRequiredService<IBackOfficeSignInManager>();
```

To `GetService` and handle if it's not found.